ucsb-cs156-s22/team03-s22-4pm-1
# Robert articletable delete

# Overview

This PR adds the delete column for the Article table so that **ONLY ADMIN** users can **delete** article entries. Logged-in users should **NOT** see these changes.

# Issues Addressed

Closes #11 

# Details

## Logged-in Users

![Screenshot_20220517_185920](https://user-images.githubusercontent.com/59896939/168941668-aaa65ef5-c773-4780-94e6-084dcef4a6c3.png)

## Admin Users

![Screenshot_20220517_185807](https://user-images.githubusercontent.com/59896939/168941550-794b7c8c-ba22-4ae5-bc4d-f36f0f8db750.png)

# Test Plan (for interactive testing) 
 
1. Login as an **admin** user and create new article entries if needed with the Swagger UI.
2. Go under `/article/list` endpoint and verify that the table shows the delete column with the delete button.
3. Logout and login as a **logged-in** user, without admin privileges.
4. Verify that the delete column **DOES NOT** show up, nor does any delete button.
5. Log back in as an **admin** and delete any entry. Verify that the entry no longer shows up in the table.
